docs/usage/cli/index.rst

```diff
@@ -0,0 +1,46 @@
+============
+augur index
+============
+
+.. argparse::
+    :module: augur
+    :func: make_parser
+    :prog: augur
+    :path: index
+
+Speed up filtering with a sequence index
+========================================
+
+As we describe in :doc:`the zika tutorial <docs.nextstrain.org:tutorials/zika_tutorial>`, augur index precalculates the composition of the sequences (e.g., numbers of nucleotides, gaps, invalid characters, and total sequence length) prior to filtering.
```

Comment:
```suggestion
As we describe in :doc:`the zika tutorial <docs.nextstrain.org:tutorials/zika>`, augur index precalculates the composition of the sequences (e.g., numbers of nucleotides, gaps, invalid characters, and total sequence length) prior to filtering.
```

I think this change will on necessary if you merge https://github.com/nextstrain/docs.nextstrain.org/pull/48 before this and haven't configured a redirect for this page (see https://github.com/nextstrain/docs.nextstrain.org/pull/48#pullrequestreview-599920542).